plugins/modules/ome_firmware_catalog.py

```diff
@@ -121,6 +121,7 @@
 notes:
     - If I(repository_password) is provided, then the module always reports the changed status.
     - Run this module from a system that has direct access to Dell OpenManage Enterprise or OpenManage Enterprise Modular.
+    - This module supports both IPv4 and IPv6 address for I(hostname).
```

Comment:
Update line 124 to suggested by Raj and generate rst.

This module supports both IPv4 and IPv6 address.